libraries/react-experimental/package.json

```diff
@@ -19,7 +19,7 @@
     "webpack": "5.70.0"
   },
   "dependencies": {
-    "react": "18.0.0",
+    "react": "0.0.0-experimental-79ed5e18f-20220217",
```

Comment:
This was erroneously updated in #1514

Reply:
Great catch! Is there a way to prevent that case happening again?

Reply:
It's tricky because in some sense it did the right thing, like, 18 > 0.0.0-... and the tests gave the same result

Reply:
Is it possible to make a test that fails if the version is unexpected?